provider/aws/lambda/autoscale/handler.go

```diff
@@ -14,9 +14,16 @@ import (
 	"github.com/aws/aws-sdk-go/service/ecs"
 )
 
+const (
+	HAInstanceCountParam   = "InstanceCount"
+	NoHaInstanceCountParam = "NoHaInstanceCount"
+)
+
 var (
 	CloudFormation = cloudformation.New(session.New(), nil)
 	ECS            = ecs.New(session.New(), nil)
+	icParam        = HAInstanceCountParam
```

Comment:
can you move this parameter inside the `autoscale` function?
seems like it's the only place it's being used